### PR TITLE
[BACKUP] Add CRR functionality for IaaSVM and other CRR commands

### PR DESCRIPTION
**Pull request by akshayneema**
**Description**
Added Cross Region Restore functionality in the CLI command for IaaSVM restore(restore-disks). Added other CRR commands for enabling CRR on a vault, list/show containers in secondary region, list/show items in secondary region, list/show RPs in secondary region, list/show jobs in secondary region, etc.  azure-mgmt-recoveryservicesbackup SDK version has been updated to 0.11.0 from 0.6.0 for the same.

**Testing Guide**

- az backup vault backup-properties set -g {rg} -n {vault} --cross-region-restore-flag {True/False}
- az backup restore restore-disks -g {rg} -v {vault} -c {container} -i {item} -r {rp} --storage-account {storage_account} -t {target_rg} --use-secondary-region
- az backup item list -g {rg} -v {vault} -c {conatiner} --backup-management-type {AzureWorkload/AzureIaasVM} --workload-type {MSSQL/SAPHANA/VM} --use-secondary-region
- az backup container show -g {rg} -v {vault} -n {conatiner} --backup-management-type {AzureWorkload/AzureIaasVM} --use-secondary-region
- az backup recoverypoint list -g {rg} -v {vault} -c {container} -i {item} --backup-management-type {AzureWorkload/AzureIaasVM} --workload-type {MSSQL/SAPHANA/VM} --use-secondary-region
- az backup job wait -g {rg} -v {vault} -n {job_id} --use-secondary-region

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
BACKUP

File: src/azure-cli/azure/cli/command_modules/backup/_params.py [hunk 1]
```diff
@@ -78,6 +78,7 @@ def load_arguments(self, _):
     with self.argument_context('backup vault backup-properties set') as c:
         c.argument('backup_storage_redundancy', arg_type=get_enum_type(['GeoRedundant', 'LocallyRedundant']), help='Sets backup storage properties for a Recovery Services vault.')
         c.argument('soft_delete_feature_state', arg_type=get_enum_type(['Enable', 'Disable']), help='Set soft-delete feature state for a Recovery Services Vault.')
+        c.argument('cross_region_restore_flag', arg_type=get_enum_type(["True", "False"]), help='Set cross-region-restore feature state for a Recovery Services Vault. Default: False.')
```
Comment (fengzhou-msft) [inline src/azure-cli/azure/cli/command_modules/backup/_params.py:81]: `cross_region_restore_flag` will be treated as a `str`, you may need to convert it to bool before you pass it to SDK.

File: src/azure-cli/azure/cli/command_modules/backup/_params.py [hunk 2]
```diff
@@ -87,9 +88,12 @@ def load_arguments(self, _):
     with self.argument_context('backup container show') as c:
         c.argument('name', container_name_type, options_list=['--name', '-n'], help='Name of the container. You can use the backup container list command to get the name of a container.')
         c.argument('backup_management_type', backup_management_type)
+        c.argument('use_secondary_region', arg_type=get_three_state_flag(), help='Use this flag to show container in secondary region. Default: false.')
```
Comment (fengzhou-msft) [inline src/azure-cli/azure/cli/command_modules/backup/_params.py:91]: This seems to be a command behavior switch, not a resource property. It's recommended to use `action='store_true'` for such cases.

File: src/azure-cli/azure/cli/command_modules/backup/custom.py [hunk 3]
```diff
@@ -23,14 +23,68 @@
...
+    cross_region_restore_cf, backup_crr_job_details_cf, crr_operation_status_cf, backup_crr_jobs_cf)
 
 logger = get_logger(__name__)
 
+# Mapping of workload type
+secondary_region_map = {"eastasia": "southeastasia",
+                        "southeastasia": "eastasia",
+                        "australiaeast": "australiasoutheast",
+                        "australiasoutheast": "australiaeast",
+                        "australiacentral": "australiacentral2",
```
Comment (fengzhou-msft) [inline src/azure-cli/azure/cli/command_modules/backup/custom.py:41]: AKS also has a similar [map](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acs/custom.py#L3004). It is the same or may be different for different RPs?

Comment (akshayneema) [reply src/azure-cli/azure/cli/command_modules/backup/custom.py:41]: No, it seems to be a different map.

File: src/azure-cli/azure/cli/command_modules/backup/_params.py (hunk 1 above)
Comment (akshayneema) [reply src/azure-cli/azure/cli/command_modules/backup/_params.py]: Did the changes. Please review once.

File: src/azure-cli/azure/cli/command_modules/backup/_params.py (hunk 2 above)
Comment (akshayneema) [reply src/azure-cli/azure/cli/command_modules/backup/_params.py]: Did the changes. Please review once.

**Review by fengzhou-msft — APPROVED** [review]

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**